docs/products/flink/concepts/supported_syntax_sql_editor.rst

```diff
@@ -0,0 +1,38 @@
+Supported syntax in the Aiven SQL editor
```

Comment:
Could this be an article about using the Aiven SQL editor? Like, can I save queries? Run them? Great!

And if there are limitations to the syntax, let's list them here but not as the main point of the post. There's good info here but perhaps we can tell the story more constructively, even without removing any of the warnings.

Reply:
True, that's now edited a bit in https://github.com/aiven/devportal/pull/267 ;)